tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+var =
+"Hello World!"
+"""
+
+
+bad_file_unexpected_comma = """
+module UnexpectedComma (..) where
+va,r =
+"Hello World!"
+"""
+
+
+good_file_no_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+100
+"""
+
+
+bad_file_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+"""
+
+
+good_file_not_empty = """
```

Comment:
IIUC, we are testing a non-empty file with some valid content. The line feed in the middle of an assignment is a part of the syntax of the elm language.